scripts/pdf/render.py

```diff
@@ -22,8 +22,7 @@ def main(loc, colorscheme):
 
     #Checking correctness of path
     if not os.path.isdir(loc):
-        print("Invalid directory. Please try again!", file = sys.stderr)
-        sys.exit(1)
+        raise ValueError("Invalid directory. Please try again!", file=sys.stderr)
```

Comment:
Not sure here. This might be semantically more correct, but by doing this we generate a Python stack trace, and the output is much more complicated - leading it to be less user-friendly.